content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
+Jak każda technologia, z którą nie jest się zaznajomionym, React ma swoją krzywą uczenia. Z odrobiną praktyki i cierpliwości, *zrozumiesz* na czym polega.
 
-### First Examples {#first-examples}
+### Podstawowe przykłady {#first-examples}
 
-The [React homepage](/) contains a few small React examples with a live editor. Even if you don't know anything about React yet, try changing their code and see how it affects the result.
+[Strona główna](/) Reacta zawiera kilka małych przykładów z edytorem kodu live. Nawet jeśli nie wiesz jeszcze nic o React, spróbuj zmienić kod i zobaczyć jakie to przyniesie efekty.
 
-### React for Beginners {#react-for-beginners}
+### React dla początkujących {#react-for-beginners}
 
-If you feel that the React documentation goes at a faster pace than you're comfortable with, check out [this overview of React by Tania Rascia](https://www.taniarascia.com/getting-started-with-react/). It introduces the most important React concepts in a detailed, beginner-friendly way. Once you're done, give the documentation another try!
+Jeśli wydaje Ci się, że dokumentacja Reacta prowadzi Cię w zbyt szybkim tempie, sprawdz [ten przegląd Reacta napisany przez Tanię Rascię](https://www.taniarascia.com/getting-started-with-react/). Wprawdzi Cię to w najistotniejsze koncepty Reacta w szczegółach, zachowując styl zrozumiały dla początkujących. Gdy będziesz gotowy, spróbuj ponownie przeczytać naszą dokumentację!
 
-### React for Designers {#react-for-designers}
+### React dla Designerów {#react-for-designers}
 
-If you're coming from a design background, [these resources](http://reactfordesigners.com/) are a great place to get started.
+Jeśli zaczynając z Reactem, przychodzisz ze środowiska graficznego, [te źródła](http://reactfordesigners.com/) mogą być świetnym miejscem do startu.
 
-### JavaScript Resources {#javascript-resources}
+### Źródła JavaScript {#javascript-resources}
 
-The React documentation assumes some familiarity with programming in the JavaScript language. You don't have to be an expert, but it's harder to learn both React and JavaScript at the same time.
+Dokumentacja Reacta zakłada znajomość języka programowania JavaScript na pewnym poziomie. Nie musisz być ekspertem, jednakże nauka zarówno Reacta jak i JavaScript w tym samym czasie może być nieco cięższa.
 
-We recommend going through [this JavaScript overview](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level. It will take you between 30 minutes and an hour but you will feel more confident learning React.
+Proponujemy zapoznanie się z [przeglądem informacji o JavaScript](https://developer.mozilla.org/pl/docs/Web/JavaScript/Ponowne_wprowadzenie_do_JavaScript), by sprawdzić swoją wiedzę. Powinno Ci to zająć około 30 minut do godziny, jednak sprawi to, że nauka Reacta będzie bardziej komfortowa.
```

Comment:
```suggestion
Proponujemy zapoznanie się z [przeglądem informacji o JavaScripcie](https://developer.mozilla.org/pl/docs/Web/JavaScript/Ponowne_wprowadzenie_do_JavaScript), by sprawdzić swoją wiedzę. Powinno Ci to zająć około 30 minut do godziny, a sprawi, że nauka Reacta będzie bardziej komfortowa.
```

Reply:
@jakubdrozdek zastanawiam się nad kwestią odmiany 'JavaScript', co prawda odmieniamy odpowiednio React przez przypadki, ale nawet tytuł podlinkowanego artykułu na Mozilli sugeruje brak odmiany. Wydaje mi sie, że kwestia do ustalenia w styleguide/glossary na później, żeby wszędzie było jednakowo.

Reply:
@iamarek Coraz bardziej skłaniam się ku temu, żeby nie inspirować się zbyt mocno MDN-em, a przynajmniej przesiewać go przez drobne sito. Jest tam całkiem sporo tekstu tłumaczonego maszynowo lub przez niedoświadczonych tłumaczy. W necie znalazłem też dużo przykładów, gdzie odmieniano Javascript jak polskie słowo (jak słowo "skrypt": Javascriptu, Javascriptem).
Jest też trzecie wyjście: można pisać "językiem Javascript", "językowi Javascript" itd.

Ja jestem za odmianą + od czasu do czasu poprzedzanie słowem "język".

Reply:
@jakubdrozdek  ok, zmieniam, proponuję dodać do słownika odmianę słowa tak samo jak z Reaktem, żeby uniknąć problemów typu JavaScriptcie - JavaScripcie - JavaSkrypcie itd.

Reply:
@iamarek Jeśli możesz, daj approve dla mojego pierwszego PR-a ze style guidem. Jak przejdzie, to dodam kolejnego na glosariusz.

Reply:
@jakubdrozdek nie wydaje mi się, żebym mógł. Nie jestem maintainerem tego repozytorium.